src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -580,6 +589,135 @@ public ExpressionStringPojo getConceptNormalForm(
 
 		return new ExpressionStringPojo(expression.toString(includeTerms));
 	}
+	
+	@PostMapping(value = "/{branch}/concepts/copy")
+	@PreAuthorize("hasPermission('AUTHOR', #branch)")
+	@JsonView(value = View.Component.class)
+	public Collection<ConceptMini> copyConcepts(
+			@ApiParam(value = "Destination branch where the concepts are copied to.")
+			@PathVariable String branch,
+			@ApiParam(value = "Source branch where the concepts are selected from.")
+			@RequestParam String sourceBranch,
+			@ApiParam(value = "ECL expression for selecting concepts to copy between the branches.")
+			@RequestParam String ecl,
+			@ApiParam(value = "Include dependant components. Defaults to false.")
+			@RequestParam(required = false, defaultValue = "false") boolean includeDependencies,
+			@RequestHeader(value = "Accept-Language", defaultValue = Config.DEFAULT_ACCEPT_LANG_HEADER) String acceptLanguageHeader) throws ServiceException {
```

Comment:
acceptLanguageHeader seems to be a redundant parameter.

Reply:
@QuyenLy87 Removed this parameter